instrumentation/spring/spring-integration-4.1/library/src/test/java/ComplexPropagationTest.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.LibraryInstrumentationExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class ComplexPropagationTest extends AbstractComplexPropagationTest {
+
+  @RegisterExtension
+  static final InstrumentationExtension testing = LibraryInstrumentationExtension.create();
+
+  public ComplexPropagationTest() {
```

Comment:
```suggestion
  ComplexPropagationTest() {
```
It seems there are many similar problems in this PR, you can address them togother.